README.md

```diff
@@ -255,19 +255,19 @@ opentok.registerCallback('connection', 'created', url, function(error, callback)
 });
 ```
 
-Note that you can only register a callback for a specific group&event. When registering a new callback for the
-same event you are replacing the previous registration.
+Note that you can only register a callback for a specific group and event. When registering
+a new callback for the a previously registered event, you replace the previous registration.
 
-To unregister a callback you have can use the ``opentok.unregisterCallback(callbackId, callback)`` method.
+To unregister a callback, use the `OpenTok.unregisterCallback(callbackId, callback) method.
```

Comment:
It really should be `OpenTok#unregisterCallback(callbackId, callback)` since dot implies static method while hash implies instance method.